### PR TITLE
feat(compass): bump up to v0.2.1

### PR DESCRIPTION
**Pull request by mabdh**

### CODE REVIEW

File: stable/compass/values.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ app:
 
   service:
     annotations:
-      {}
+      - projectcontour.io/upstream-protocol.h2c: 80,http
```
Comment (StewartJingga) [inline stable/compass/values.yaml:29]: CMIIW but annotations is expecting a map.

File: stable/compass/values.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@ app:
 
   service:
     annotations:
-      {}
+      projectcontour.io/upstream-protocol.h2c: 80,http
```
Comment (rohilsurana) [inline stable/compass/values.yaml:29]: Can we avoid having this in default values? This is a setup specific annotation and can be added at the time of deployment.

Comment (StewartJingga) [reply stable/compass/values.yaml:29]: I guess that makes sense. cc: @mabdh

**Review by StewartJingga — APPROVED** [review]

File: stable/compass/values.yaml (hunk 2 above)
Comment (mabdh) [reply stable/compass/values.yaml]: updated thanks guys @rohilsurana @StewartJingga

**Pull request merged**